ex_cubic_ods_ingestion/lib/ex_cubic_ods_ingestion/oban_ingest_worker_error.ex

```diff
@@ -0,0 +1,35 @@
+defmodule ExCubicOdsIngestion.ObanIngestWorkerError do
+  @moduledoc """
+  Provides a way for Oban to update status of load upon a complete failure of an 'ingest' job.
+  """
+
+  alias ExCubicOdsIngestion.Schema.CubicOdsLoad
+
+  @spec handle_event(
+          :telemetry.event_name(),
+          :telemetry.event_measurements(),
+          :telemetry.event_metadata(),
+          :telemetry.handler_config()
+        ) :: [CubicOdsLoad.t()]
+  @doc """
+  Matches on the Ingest worker, and updates the status of load once all attempts have failed.
+  """
+  def handle_event(
+        [:oban, :job, :exception],
+        _measure,
+        %{worker: "ExCubicOdsIngestion.Workers.Ingest"} = meta,
```

Comment:
To avoid the else, you can do an additional match up here:
```suggestion
        %{worker: "ExCubicOdsIngestion.Workers.Ingest", attempt: attempts, max_attempts: attempts} = meta,
```